vertica/assets/configuration/spec.yaml

```
@@ -32,6 +32,35 @@ files:
             value:
               type: integer
               example: 5433
+          - name: include_tables
+            required: true
+            description: The tables to include for metric collection.
```

Should we include a note that the exclude options will take precedence over the include list?

```suggestion
            description: Enable metric groups for metric collection. Metric groups are based on Vertica tables. By default all metric groups are collected.
```